users/cloudformation.yml

```diff
@@ -14,21 +14,63 @@ Parameters:
     Type: String
 
 
+Mappings:
+
+  SsmParameters:
```

Comment:
Minor: sms (I'm fine leaving this as is)

Reply:
An SSM Parameter is a parameter in the Simple Systems Management parameter store

Reply:
I stand corrected